.env.sample

```diff
@@ -2,4 +2,5 @@ ENVIRONMENT=local
 APP_NAME=seedling
 LOCAL_APP_PORT=8050
 LOCAL_MSSQL_PORT=1433
-LOCAL_IMAGE_NAME=flowehr_seedling_app
\ No newline at end of file
+LOCAL_IMAGE_NAME=flowehr_seedling_app
+APPLICATIONINSIGHTS_CONNECTION_STRING=application_insight_connection_string
```

Comment:
I think helps when you are testing/viewing the logs reflected in application insights.

Reply:
> I don't think `APPLICATIONINSIGHTS_CONNECTION_STRING` should be set for local, but apart from that – LGTM!
> 
> I would have
> 
> ```python
> if os.environ.get("ENVIRONMENT", default="dev") == "local":
>     return
> ```
> 
> in initialise_logging

I think it helps when you are testing/viewing the logs in application insights. Also if is not set it doesn't block or error.

Reply:
indeed it does help when app insights is deployed! but in `ENVIRONMENT=local` and calling `make serve-local` this is not the case. Therefore I don't think it makes sense to have `APPLICATIONINSIGHTS_CONNECTION_STRING` defined in the sample .env file. If there is a scenario where you want to hook your local app up to a cloud deployed cosmos/mssql/appinsights then there should be a separate sample .env file (i.e. `.env.cloud.sample` or something) and a new `make` target for that, in my opinion at least!
@jjgriff93 / @martinpeck  what do you think?

Reply:
it's benign if not present
i.e. nothing fails...it just does't log to App Insights

Reply:
Oh....I see...the local config.
It's not important, so it can be pulled from the config.

If you want to test that your logging is going into App Insights then someone can add that config directly.
For the most part, a local developer won't care.
It's only someone "developing the logging" that will want this, and that person can happily set a local env var themselves.

Reply:
Done d94cd28bc548811dd564f2239563d546abeabd1a